README.md

```diff
@@ -7,6 +7,10 @@ The golang bindings for libtoxcore
 
 ### Installation
 
+    mkdir -p $(go env GOPATH)/github.com/kitech
+    ( cd $(go env GOPATH)/github.com/kitech; git clone https://github.com/TokTok/go-toxcore-c.git go-toxcore )
+    # fetch libtoxcore if necessary
+    # see https://github.com/irungentoo/toxcore/blob/master/INSTALL.md
     go get github.com/kitech/go-toxcore
```

Comment:
Why not change this to the new URL?

Reply:
I wasn't sure what the correct canonical URL was - is it this repository? The earlier `cd` commands place this repository in the old path.

Reply:
@kitech thoughts?

As far as I understand, we'll have the toktok repo as the canonical one for the core API. kitech wants to work on a higher level API. TokTok will maintain the low level one.